backend/core/types/src/backend-swagger.ts

```diff
@@ -5398,6 +5428,35 @@ export interface UnitCreate {
   bmrProgramChart?: boolean
 }
 
+export interface AddressCountyRequiredCreate {
```

Comment:
I'm leaning to delete the remainder of the backend-swagger changes below as well but will look to confirmation from @ludtkemorgan here

Reply:
I think the generate script just wasn't run after some of the last features added by the google team. Lets keep all of this to keep it in sync with the DTOs. we'll do a bit of cleanup when we recreate this in prisma backend